src/resolver/nodeModuleLookup.ts

```diff
@@ -92,7 +109,24 @@ export interface INodeModuleLookup {
 }
 
 export function nodeModuleLookup(props: IResolverProps, parsed: IModuleParsed): INodeModuleLookup {
-  const folder = findTargetFolder(props, parsed);
+  let folder = findTargetFolder(props, parsed);
+  let packageJSONFile: string;
+  let useLocalField = false;
+  if (!folder) {
+    folder = findTargetFolderInTypescriptPaths(props);
```

Comment:
I think this isn't gonna work, it will never hit it. Since "local:main" is used against local module and never against the real ones. Since monorepo structure ends up being like a normal projects with replaced paths that look like packages but reference local files instead. We're adding an additional overhead here.

Reply:
I wrote this as I ran into errors.  It did end up fixing the issue, but now that I understand the repo more, it's out of place.  I think "sibling modules" should probably be found after attempting pathLookup and before moduleLookup.

Or perhaps they should be part of pathLookup?